### DISCUSSION TITLE
color theme wrong for gnome windows

### DISCUSSION THREAD

**Discussion by zupd** [regolith-linux/regolith-desktop#978] [Q&A]
After a restart (a while after I had installed regolith 3.0 on ubuntu) I noticed that the nord theme no longer applied to the gnome windows.

Now the gnome windows has the default dark color instead of the theme applied by regolith-look-nord.

Other parts of the desktop got the correct colors. So I am not sure what's wrong. Has gnome managed to apply a dark theme over the nord theme and overwritten it or something? Do you have any suggestions to what I can do to check what the problem is?

**Comment by zupd** [comment]
I am using Regolith on Wayland and Ubuntu 23.04

**Reply by kgilmer** [reply]
I believe this is "by design" of GTK 4.  I am not aware of a way to cause GTK 3 themes to work in GTK 4.

**Reply by kgilmer** [reply]
FYI here's what Regolith does to set the GTK theme: https://github.com/regolith-linux/regolith-look-default/blob/main/usr/share/regolith-look/default_loader.sh#L15

**Reply by zupd** [reply]
Thanks. Looks like the theme is updated for GTK 4 too:
https://github.com/EliverLara/Nordic

But I think regolith hasnt updated the theme yet

**Reply by zupd** [reply]
Ended up using a quickfix for now: https://github.com/odziom91/libadwaita-theme-changer to change the adwaita theme to nordic gtk4
